.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     sourceType: 'module',
   },
   rules: {
+    'no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
```

Comment:
This rule allows us to mark unused vars (if we want to) as `_someUnusedVar`. There is an example below in one of the Cypress files.